src/components/WalletImageSlider.tsx

```diff
@@ -10,6 +10,17 @@ export default function WalletImageSlider({
   sliderValue,
   setSliderValue,
 }: SliderImageWalletProps) {
+  const SliderValueEqualsIndex = (index: number) => {
+    if (sliderValue >= index) {
+      return true;
+    }
```

Comment:
Could turn this into a 1 liner
```suggestion
    if (sliderValue >= index) return true;
```

Reply:
addressed.